rllib/policy/eager_tf_policy.py

```diff
@@ -293,10 +283,16 @@ def __init__(self, observation_space, action_space, config):
             #  Just like torch Policy does.
             self._optimizer = optimizers[0] if optimizers else None
 
+            self._initialize_loss_from_dummy_batch(
```

Comment:
Moved this here for consistency (same behavior as TorchPolicy). Also fixes a problem with curiosity where we do need the optimizer before loss init.